### PR TITLE
[EdgeDB] ProgressSummary schema

### PR DESCRIPTION
**Pull request by andrewmurraydavid**
[Monday Task](https://seed-company-squad.monday.com/boards/5989610236/pulses/5792959432)

┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/6200458679) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by bryanjnelson — APPROVED** [review]
LGTM!

File: dbschema/progress-summary.esdl
Comment (CarsonF) [inline dbschema/progress-summary.esdl:2]: I can't help but feel that this shouldn't be in the default module.
Maybe
```edgeql
ProgressReport::ProgressSummary
# or
ProgressReport::ProductProgress::Summary
```
Thoughts @andrewmurraydavid @bryanjnelson?

File: dbschema/progress-summary.esdl
Comment (CarsonF) [inline dbschema/progress-summary.esdl:10]:
```suggestion
    totalVerseEquivalents: int32;
```

File: dbschema/progress-report.esdl
Comment (CarsonF) [inline dbschema/progress-report.esdl:12]: I can't see this being that helpful. I don't think app code will use it. Nor will adhoc exploration benefit.

Adhoc exploration would probably be looking for the cumulative summary.
I'd be ok with that
```edgeql
single summaryCumulative := (select .<report[is ProgressSummary] filter .period = ProgressSummary::Period.Cumulative)
```

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: dbschema/progress-summary.esdl
Comment (andrewmurraydavid) [reply dbschema/progress-summary.esdl:10]: Max 16 is 32k and number of all verses is 31k. Curious why int32 here.

File: dbschema/progress-report.esdl
Comment (andrewmurraydavid) [reply dbschema/progress-report.esdl:12]: I thought that you wanted me to backfill manually based on our conversation on slack yesterday. I am ok removing it.

File: dbschema/progress-summary.esdl
Comment (CarsonF) [reply dbschema/progress-summary.esdl:10]: Actually `float32`.
Max verse equivalents (for full bible) is 35683.8125)

File: dbschema/progress-report.esdl
Comment (CarsonF) [reply dbschema/progress-report.esdl:12]: Sorry. I was just answering on the "automatic backfill" question. Works for me. We can always add/change later too.

File: dbschema/progress-summary.esdl
Comment (bryanjnelson) [reply dbschema/progress-summary.esdl:2]: Do we have a `ProductProgress` now?  

Ah, yes, Brent has a PR up for it.  I believe I like `ProgressReport::ProductProgress::Summary` then.

Comment (andrewmurraydavid) [reply dbschema/progress-summary.esdl:2]: Should it move to a separate file or can it stay here, just changing the namespace?

Comment (bryanjnelson) [reply dbschema/progress-summary.esdl:2]: Hmm...I could see moving it [over here](https://github.com/SeedCompany/cord-api-v3/pull/3091/files) maybe, after this is solidified?  I'll let @CarsonF weigh in.

Comment (andrewmurraydavid) [reply dbschema/progress-summary.esdl:2]: That's what I was thinking too, @bryanjnelson . That's why I suggested to wait until #3091 is merged.

Comment (bryanjnelson) [reply dbschema/progress-summary.esdl:2]: Ah...must've missed your edit!

**Review by CarsonF — APPROVED** [review]
Thanks @andrewmurraydavid!

**Comment by andrewmurraydavid** [comment]
Thanks for bring it home, @CarsonF !

**Pull request merged**
